emmy.go

```diff
@@ -19,16 +19,26 @@ package main
 
 import (
 	"os"
+	"time"
 
 	"github.com/urfave/cli"
 	emmy "github.com/xlab-si/emmy/cmd"
 )
 
+// version marks the version of emmy.
+// Its value should be set externally at compile time, by appending
+// -ldflags "-X main.version=x.y.z" option to go build/run/install commands at compile time.
+// In case its value remains empty, the CLI simply contains no version information.
+var version string
+
 // main runs the emmy CLI app.
 func main() {
 	app := cli.NewApp()
 	app.Name = "emmy"
-	app.Version = "0.1"
+	app.Copyright = `(c) 2017 XLAB d.o.o.
+		Licensed under Apache License, Version 2.0.`
+	app.Version = version
+	app.Compiled = time.Now()
```

Comment:
Doesn't setting the compiled time to time.Now() just mean it will be set to the current system time whenever the program is executed? The cli utility that is used in Emmy sets the right time by default when the struct App is created, as seen [here](https://github.com/urfave/cli/blob/master/app.go#L103).

Reply:
You're right. Removed.